src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary_tab.cljs

```diff
@@ -271,6 +321,10 @@
      [AclEditor {:workspace-id (:workspace-id props)
                  :dismiss-self #(swap! state dissoc :editing-acl?)
                  :update-owners #(swap! state update-in [:server-response :workspace] assoc "owners" %)}])
+   (when (:cloning? @state)
+     (common/scroll-to-top 100)
```

Comment:
I don't think we'd want to scroll on every re-render of this component (even though it probably works right now). Suggest moving this to WorkspaceCloner:component-did-mount.